README.md

```diff
@@ -1 +1,431 @@
+* [Primitives](#primitives)
+  * [Common Primitives](#common-primitives)
+  * [Declaring A Primitive Variable](#declaring-a-primitive-variable)
+  * [Redefining A Primitive Variable](#redefining-a-primitive-variable)
+  * [Printing A Primitive Variable](#printing-a-primitive-variable)
+* [Operators](#operators)
+  * [Mathematical Operators](#mathematical-operators)
+  * [Assignment Operators](#assignment-operators)
+  * [Comparison Operators](#comparison-operators)
+  * [Operation Precedence](#operation-precedence)
+* [Calling Methods](#calling-methods)
+  * [Method Call Syntax](#method-call-syntax)
+  * [Method Call Anatomy](#method-call-anatomy)
+* [NSString](#nsstring)
+  * [String Literal](#string-literal)
+  * [Introspection Methods](#introspection-methods)
+  * [Comparing Strings](#comparing-strings)
+  * [Case Change Methods](#case-change-methods)
+  * [Interpolation And Concatenation Methods](#interpolation-and-concatenation-methods)
+
+
+## NSLog Basics
+
+### String Literal
+
+```objc
+NSLog(@"Hello, World!");
+```
+
+This will print:
+
+```
+Hello, World!
+```
+
+### Format String
+
+```objc
+NSLog(@"%@, %@!", @"Hello", @"World");
+NSLog(@"There are %li lights.", 4);
+```
+This will print:
+
+```
+Hello, World!
+There are 4 lights.
+```
+
+### Format Specifiers
+
+| Type         | Format Specifier |
+|:------------:|:----------------:|
+| `NSString`   | `%@`
+| `NSInteger`  | `%li`            |
+| `NSUInteger` | `%lu`            |
+| `CGFloat`    | `%f` or `%.nf`, *where* `n` *is the number of decimal places to display* |
+| `BOOL`       | `%d`             |
+
+Examples:
+
+```objc
+NSLog(@"%@", @"Welcome!");
+NSLog(@"-1: %li", -1);
+NSLog(@"42: %lu", 42);
+NSLog(@"pi: %f, 3.141593);
+NSLog(@"sqrt2: %.12f", 1.41421356237);
+NSLog(@"yes: %d", YES);
+```
+These will print:
+
+```
+Welcome!
+-1: -1
+42: 42
+pi: 3.141593
+sqrt2: 1.414213562370
+yes: 1
+```
+
+[Back to Top](#table-of-contents)
+## Primitives
+
+### Common Primitives
+
+| Primitive   | Description |
+|:-----------:|:------------|
+| `NSInteger` | An integer value that can be either positive or negative. |
+| `NSUInteger`| An integer value that can **only** be positive (the "U" means "unsigned"). |
+| `CGFloat`   | A "float" value that can (imperfectly) hold a decimal point. **Note:** *Do not use float values to hold currency.* |
+| `BOOL`      | A `YES` or `NO` value: `0` means "no", `1` means "yes" |
+
+### Declaring A Primitive Variable
+
+Syntax:
+
+```objc
+Type variableName = value;
+```
+
+Examples:
+
+```objc
+NSInteger i = 0;
+NSUInteger u = 0;
+CGFloat f = 0.0;
+BOOL hidden = NO;
+```
+**Reminder:** *Do not use a* `*` *when declaring primitives. Entering* `NSInteger *variable` *will cause the compiler to generate an* `invalid integer to pointer conversion` *warning. The same goes for declaring any of the data types. This syntax does serve a purpose so the language permits it, but your application won't do what you expect.*
```

Comment:
Same thing as in the other instance of this... "Assigning an integer to an `NSInteger *` variable will cause..."